helios/models.py

```diff
@@ -765,64 +765,61 @@ def itervoters(self):
 
     for voter_fields in reader:
       # bad line
-      if len(voter_fields) < 1:
+      if len(voter_fields) < 2:
         continue
-    
-      return_dict = {'voter_id': voter_fields[0].strip()}
 
-      if len(voter_fields) > 1:
-        return_dict['email'] = voter_fields[1].strip()
-      else:
-        # assume single field means the email is the same field
-        return_dict['email'] = voter_fields[0].strip()
+      return_dict = {'voter_type': voter_fields[0].strip(), 'voter_id': voter_fields[1].strip()}
```

Comment:
I think we need an extra check here that `voter_type` is one of the existing voter types, otherwise this could be garbage that then leads to confusion (or possibly some unexplored security issue.)

Can you add one validation step here?

Reply:
throwing an exception here results in the message shown to the end user being less rather than more specific.

(- your CSV file could not be processed. Please check that it is a proper CSV file.)

will have to think on how to do this.

Reply:
I think it's cleaner to do all validation in voter_file and then to pass the error message in the Exception up verbatim in the UI, rather than separately checking in model & in view. what do you think?